### DISCUSSION TITLE
React/svg/

### DISCUSSION THREAD

**Discussion by giscus** [yeopto/yeopto.github.io#14] [Comments]
# React/svg/

페이먼츠 미션에서 svg 사용기

https://yeopto.github.io/React/svg/

**Comment by jw-r** [comment]
좋은글 잘 보고 갑니다!

**Comment by Gilpop8663** [comment]
png 형식의 이미지도 svg로 표시할 수 있는 지 처음 알았네요! 잘 보고 갑니다 엽토!
